ueber_uns/gremien.html

```diff
@@ -29,18 +35,24 @@ <h4>Lena Linhoff</h4>
 
 <div class="card mb-3">
   <div class="card-body text-center">
-    <h3 class="card-title">Geschäftsführung, Mitgliederverwaltung und Ressourcenverwaltung</h3>
+    <h3 class="card-title">Koordination, Mitgliederverwaltung und Ressourcenverwaltung</h3>
```

Comment:
Das hier habt ihr im MR verschwiegen. Das sollte erstmal in der MV abgesegnet werden, wenn ich das richtig sehe.

Reply:
@Kevin2 @hmoldenhauer

Reply:
Ja, da hast du recht. Wir hatten Marie gefragt, ob wir das schon ändern sollten, sie hat gesagt, dass wir das schon mal machen sollen.

Aber natürlich müssen wir uns das noch mal von der MV genehmigen lassen. Wenn du willst, ändere ich das noch zurück.

Reply:
Namensänderungen dieser generischen VPs? Ich denke das geht in Ordnung.